### ISSUE TITLE
Cannot add more than 10 feeds

### ISSUE THREAD

**Issue by tensor5** [jangernert/FeedReader#366]
Hi,
I added 10 local feeds to FeedReader and when I try to add a new one it overwrites the last one.

Thanks

**Comment by bil-elmoussaoui** [comment]
Which version of FeedReader are you using?

**Comment by tensor5** [comment]
2.0

**Comment by jangernert** [comment]
And which backend are you using? I'm guessing local RSS?
Can you post the log of feedreader-daemon as you try to add another new feed?

**Comment by tensor5** [comment]
Yes, I'm using local RSS. This is the log before (last feed is https://github.com/atom/electron/releases.atom):

```
[ INFO  ] Everything is logged. Used to debug!
[ INFO  ] daemon: Notification actions supported
[ INFO  ] FeedReader Daemon 2.0
[ DEBUG ] Opening Database: /home/nicola/.local/share/feedreader/data/feedreader-04.db
[ DEBUG ] database already initialized
[ DEBUG ] database already initialized
[ DEBUG ] daemon: constructor
[ DEBUG ] daemon: new FeedServer and login
[WARNING] feedserver.unloadPlugin: no plugin loaded
[ DEBUG ] feedserver: load plugin "local"
[ DEBUG ] feedserver: engine load Local RSS Backend
[ DEBUG ] feedserver: plugin loaded Local RSS Backend
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] CachedActionManager - executeActions: no actions to perform
[WARNING] resetCachedActions
[ DEBUG ] daemon: login status = FEED_READER_LOGIN_RESPONSE_SUCCESS
[ DEBUG ] daemon: bus aquired
[ DEBUG ] daemon: symbolicIcon
[ DEBUG ] feedserver: symbolicIcon
[ DEBUG ] SELECT count(*) FROM main.feeds
[ DEBUG ] count 9
[ INFO  ] addFeed: ID = feedID10
[ DEBUG ] Utils: downloadIcon() url = "https://github.com/electron/electron/releases"
[ DEBUG ] last clean: 2017-01-28 21:56:04
[ DEBUG ] now: 2017-02-14 18:24:08
[ DEBUG ] difference: 16.000000
[ DEBUG ] Daemon: checkOnline
[ DEBUG ] Ping: https://duckduckgo.com/
[ DEBUG ] Ping: modified URL duckduckgo.com
[ DEBUG ] Ping: trying ip-address 176.34.131.233
[ DEBUG ] HTTP/1.1 200 OK
[ DEBUG ] Ping: success!
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] CachedActionManager - executeActions: no actions to perform
[WARNING] resetCachedActions
[ INFO  ] daemon: sync started
[ DEBUG ] Ping: https://duckduckgo.com/
[ DEBUG ] Ping: modified URL duckduckgo.com
[ DEBUG ] Ping: trying ip-address 176.34.131.233
[ DEBUG ] HTTP/1.1 200 OK
[ DEBUG ] Ping: success!
[WARNING] dbDaemon: Deleting articles without feed
[WARNING] dbDaemon: Deleting unsibscribed feeds
[ DEBUG ] daemon: symbolicIcon
[ DEBUG ] feedserver: symbolicIcon
[ DEBUG ] getArticles for feed: Phoronix
[ DEBUG ] getArticles for feed: Recent additions
[ DEBUG ] getArticles for feed: OSNews
[ DEBUG ] getArticles for feed: Release notes from go-ethereum
[ DEBUG ] getArticles for feed: Release notes from black-screen
[ DEBUG ] getArticles for feed: Release notes from solidity
[ DEBUG ] getArticles for feed: Release notes from atom
[ DEBUG ] getArticles for feed: Release notes from apm
[ DEBUG ] getArticles for feed: Release notes from parity
[ DEBUG ] getArticles for feed: Release notes from electron
[ ERROR ] dbErrorLog: 21: misuse at line 361 of [a65a62893c]
[ DEBUG ] Opening Database: /home/nicola/.local/share/feedreader/data/feedreader-04.db
[ DEBUG ] localInterface: 179 articles written
[ DEBUG ] FeedServer: new articles: 10
[ DEBUG ] FeedServer: grabContent
[ DEBUG ] Grabber: process article: /ethereum/go-ethereum/releases/tag/v1.5.0
[ DEBUG ] Grabber: no config (.txt) found for article: /ethereum/go-ethereum/releases/tag/v1.5.0
[ DEBUG ] Grabber: no config (.txt) - cutSubdomain - found for article: /ethereum/go-ethereum/releases/tag/v1.5.0
```

Then I add https://github.com/electron/asar/releases.atom, and after a while:

```
[ INFO  ] Everything is logged. Used to debug!
[ INFO  ] daemon: Notification actions supported
[ INFO  ] FeedReader Daemon 2.0
[ DEBUG ] Opening Database: /home/nicola/.local/share/feedreader/data/feedreader-04.db
[ DEBUG ] database already initialized
[ DEBUG ] database already initialized
[ DEBUG ] daemon: constructor
[ DEBUG ] daemon: new FeedServer and login
[WARNING] feedserver.unloadPlugin: no plugin loaded
[ DEBUG ] feedserver: load plugin "local"
[ DEBUG ] feedserver: engine load Local RSS Backend
[ DEBUG ] feedserver: plugin loaded Local RSS Backend
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] CachedActionManager - executeActions: no actions to perform
[WARNING] resetCachedActions
[ DEBUG ] daemon: login status = FEED_READER_LOGIN_RESPONSE_SUCCESS
[ DEBUG ] daemon: bus aquired
[ DEBUG ] daemon: symbolicIcon
[ DEBUG ] feedserver: symbolicIcon
[ DEBUG ] SELECT count(*) FROM main.feeds
[ DEBUG ] count 10
[ INFO  ] addFeed: ID = feedID10
[ DEBUG ] last clean: 2017-01-28 21:56:04
[ DEBUG ] now: 2017-02-14 18:27:01
[ DEBUG ] difference: 16.000000
[ DEBUG ] Daemon: checkOnline
[ DEBUG ] Ping: https://duckduckgo.com/
[ DEBUG ] Ping: modified URL duckduckgo.com
[ DEBUG ] Ping: trying ip-address 176.34.131.233
[ DEBUG ] HTTP/1.1 200 OK
[ DEBUG ] Ping: success!
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] SELECT count(*) FROM main.CachedActions
[ DEBUG ] count 0
[ DEBUG ] CachedActionManager - executeActions: no actions to perform
[WARNING] resetCachedActions
[ INFO  ] daemon: sync started
[ DEBUG ] Ping: https://duckduckgo.com/
[ DEBUG ] Ping: modified URL duckduckgo.com
[ DEBUG ] Ping: trying ip-address 54.229.105.203
[ DEBUG ] HTTP/1.1 200 OK
[ DEBUG ] Ping: success!
[WARNING] dbDaemon: Deleting articles without feed
[WARNING] dbDaemon: Deleting unsibscribed feeds
[ DEBUG ] daemon: symbolicIcon
[ DEBUG ] feedserver: symbolicIcon
[ DEBUG ] getArticles for feed: Phoronix
[ ERROR ] localInterface.getArticles: Could not parse data contents
[ DEBUG ] getArticles for feed: Recent additions
[ DEBUG ] getArticles for feed: OSNews
[ DEBUG ] getArticles for feed: Release notes from go-ethereum
[ DEBUG ] getArticles for feed: Release notes from black-screen
[ DEBUG ] getArticles for feed: Release notes from solidity
[ DEBUG ] getArticles for feed: Release notes from atom
[ DEBUG ] getArticles for feed: Release notes from apm
[ DEBUG ] getArticles for feed: Release notes from parity
[ DEBUG ] getArticles for feed: Release notes from asar
[ ERROR ] dbErrorLog: 21: misuse at line 361 of [a65a62893c]
[ DEBUG ] Opening Database: /home/nicola/.local/share/feedreader/data/feedreader-04.db
[ DEBUG ] localInterface: 147 articles written
[ DEBUG ] FeedServer: new articles: 10
[ DEBUG ] FeedServer: grabContent
[ DEBUG ] Grabber: process article: /ethereum/go-ethereum/releases/tag/v1.5.0
[ DEBUG ] Grabber: no config (.txt) found for article: /ethereum/go-ethereum/releases/tag/v1.5.0
[ DEBUG ] Grabber: no config (.txt) - cutSubdomain - found for article: /ethereum/go-ethereum/releases/tag/v1.5.0
```

**Comment by jangernert** [comment]
Problem was `max(feed_id)` thought feedID9 > feedID10
Fix will be in 2.0.1.

**Comment by tensor5** [comment]
Thanks!
